tests/integration/dialog-extension/dialog_events.js

```diff
@@ -117,20 +160,24 @@ asyncTest( "dialog element with data-theme", function() {
 			var dialog = $( "#dialog-b" );
 
 			// Assert dialog theme inheritance (issue 1375):
-			ok( dialog.hasClass( "ui-page-theme-e" ), "Expected explicit theme ui-page-theme-e" );
-			ok( !$.mobile.pageContainer.hasClass( "ui-overlay-b" ), "Expected no overlay theme ui-overlay-b" );
-			ok( $.mobile.pageContainer.hasClass( "ui-overlay-a" ), "Expected default overlay theme ui-overlay-a" );
-			ok( dialog.find( ":jqmData(role=header)" ).hasClass( "ui-bar-inherit" ), "Expected header to inherit from dialog" );
-			ok( dialog.find( ":jqmData(role=content)" ).hasClass( "ui-body-e" ), "Expect content to inherit from data-theme" );
-			ok( dialog.find( ":jqmData(role=footer)" ).hasClass( "ui-bar-inherit" ), "Expected footer to inherit from dialog" );
+			assert.hasClasses( dialog[ 0 ], "ui-page-theme-e",
+				"Expected explicit theme ui-page-theme-e" );
+			assert.lacksClasses( $( "body" )[ 0 ], "ui-overlay-b",
+				"Expected no overlay theme ui-overlay-b" );
+			assert.hasClasses( $( "body" )[ 0 ], "ui-overlay-a",
```

Comment:
I cannot replace the last two assertions by a single ```.hasClassesStrict()``` because the page container may or may not have additional classes such as ```ui-viewport``` added by the page container widget. Ascertaining the presence/absence of that class is beyond the scope of this dialog-related test.